crawler/rabbit/messages/create_plate_message.py

```diff
@@ -31,6 +32,19 @@ class MessageField(NamedTuple):
     value: Any
 
 
+@dataclass
+class CreatePlateError:
+    origin: str
+    description: str
+    long_description: Optional[str] = None
+    sample_uuid: Optional[str] = None
+    field: Optional[str] = None
+
+    @property
+    def longest_description(self):
+        return self.long_description or self.description
```

Comment:
This was so that the description going to Lighthouse UI could be more descriptive of the problem while the one going to Prasanth would be more succinct as it was accompanied by the other information in the other error fields.  Now that we're thinking of adding an error type, I think it could be beneficial to remove the long_description and just make every description be detailed.